src/main/java/teammates/common/datatransfer/FeedbackSessionAttributes.java

```diff
@@ -205,13 +206,14 @@ public String getJsonString() {
         // Skip time frame checks if session type is private.
         if (this.isPrivateSession()) { return errors; }
 
-        error = validator.getValidityInfoForNonNullField("submission opening time", startTime);
+        error = validator.getValidityInfoForNonNullField(FieldValidator.START_TIME_FIELD_NAME, startTime);
         if (!error.isEmpty()) { errors.add(error); }
 
-        error = validator.getValidityInfoForNonNullField("submission closing time", endTime);
+        error = validator.getValidityInfoForNonNullField(FieldValidator.END_TIME_FIELD_NAME, endTime);
         if (!error.isEmpty()) { errors.add(error); }
 
-        error = validator.getValidityInfoForNonNullField("time for the responses to become visible", resultsVisibleFromTime);
+        error = validator.getValidityInfoForNonNullField(FieldValidator.RESULTS_VISIBLE_TIME_FIELD_NAME,
+                                                         resultsVisibleFromTime);
```

Comment:
I replaced magic strings that are already defined (e.g., "submission opening time"). If this is okay, I'll proceed to define new constants for the remaining magic strings (e.g., "creator's email").